vtex/admin-ui
# feat(i18n): adjust translations to follow VTEX localization pattern

#### What is the purpose of this pull request?


As the title says.

[It implements the specs defined on this RFC](https://github.com/vtex/admin-ui/discussions/579)

* Move translations to the `/messages` folder
* Setup the crowdin config file

Next Steps:
* Add the I18nProvider to the ThemeProvider and add the locale prop.

#### How should this be manually tested?


You can check the components stories to check if the i18n works as it should. [Search example](https://620e870c28c4ae003a705d42-ocvmlcgbke.chromatic.com/?path=/story/admin-ui-review-search--playground&args=locale:ko-KR)

#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)
